### DISCUSSION TITLE
OpenVino Development Tool Installation on Raspberry always fail

### DISCUSSION THREAD

**Discussion by mabusharkh** [openvinotoolkit/openvino#19349] [Q&A] [answered]
Dear All,

I am new to this topic so please excuse my ignorance. 

I have been trying to install OpenVino development tools on Raspi 3 with all possible means since the last 2 weeks and I did follow all the steps line by line as per the article https://docs.openvino.ai/2023.0/openvino_docs_install_guides_install_dev_tools.html#install-dev-tools and it always fail by saying:

_ERROR: Could not find a version that satisfies the requirement openvino==2022.3.1 (from openvino-dev[tensorflow2])
ERROR: No matching distribution found for openvino==2022.3.1_

I am using the command: pip install openvino-dev but it is not working. I tried different versions of OpenVino hoping that this will resolve the OpenVino development installation issue but it did not work.

how about if I downloaded the installation package [l_openvino_toolkit_debian9_2023.1.0.dev20230811_armhf.tgz](https://storage.openvinotoolkit.org/repositories/openvino/packages/master/2023.1.0.dev20230811/l_openvino_toolkit_debian9_2023.1.0.dev20230811_armhf.tgz) 

but in this case how I am going to install it? I appreciate any help please?

Thanks
Mohd

**Comment by ilya-lavrenov** [comment, marked as answer]
Hi @mabusharkh 
We don't currently provide `openvino` and `openvino-dev` pip packages.

Please, use OpenVINO archive (like [l_openvino_toolkit_debian9_2023.1.0.dev20230811_armhf.tgz](https://storage.openvinotoolkit.org/repositories/openvino/packages/master/2023.1.0.dev20230811/l_openvino_toolkit_debian9_2023.1.0.dev20230811_armhf.tgz) which you mentioned), call setupvars.sh and use OpenVINO Model Converter from there. 

But since Raspberry PI board is targeted for deployment, it's recommended to prepare your model in offline on more powerful x64 / arm64 CPU and then transfer your model to the device only for inference.
